Userland/Libraries/LibWasm/Printer/Printer.cpp

```diff
@@ -541,33 +541,6 @@ void Printer::print(Wasm::Module const& module)
     print(")\n");
 }
 
-void Printer::print(Wasm::Module::Function const& func)
```

Comment:
_something_ should print the functions' contents.

Reply:
I removed `Function` as a whole

Reply:
Ah, that's true, it's being printed through `code.func()`.